test/pkg/environment/azure/environment.go

```diff
@@ -45,11 +45,11 @@ type Environment struct {
 
 func NewEnvironment(t *testing.T) *Environment {
 	env := common.NewEnvironment(t)
-
-	return &Environment{
-		Region:      "westus2",
+	azureEnv := &Environment{
 		Environment: env,
+		Region:      "westus2",
```

Comment:
This should come from an env var rather than a hardcoded value.

Reply:
... or be auto-detected (if possible)